src/main/java/spireMapOverhaul/zones/brokenSpace/relics/BrokenDataDisk.java

```diff
@@ -0,0 +1,66 @@
+package spireMapOverhaul.zones.brokenSpace.relics;
+
+import com.evacipated.cardcrawl.mod.stslib.relics.OnChannelRelic;
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.common.ApplyPowerAction;
+import com.megacrit.cardcrawl.actions.common.RelicAboveCreatureAction;
+import com.megacrit.cardcrawl.actions.defect.ChannelAction;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.orbs.AbstractOrb;
+import com.megacrit.cardcrawl.powers.FocusPower;
+import com.megacrit.cardcrawl.relics.DataDisk;
+
+import static spireMapOverhaul.util.Wiz.adp;
+
+public class BrokenDataDisk extends BrokenRelic implements OnChannelRelic {
+    public static final String ID = "BrokenDataDisk";
+    public static final int AMOUNT = 1;
+    public boolean shouldTrigger = true;
+    private boolean usedThisTurn = false;
+
+    public BrokenDataDisk() {
+        super(ID, RelicTier.SPECIAL, LandingSound.CLINK, DataDisk.ID);
+    }
+
+    @Override
+    public void onChannel(AbstractOrb abstractOrb) {
+        if (!shouldTrigger || usedThisTurn) {
```

Comment:
What's the reason you need both `shouldTrigger` and `usedThisTurn`? Looks to me like only having `usedThisTurn` would be enough.